### PR TITLE
Check Satellite Version instead of Upgrading again

### PR DESCRIPTION
**Pull request by jyejare**
This PR has following additions and updates:
1. New functions for discovering the current satellite version
2. Updates in process of upgrade if current_version is same as To_Version then upgrade on satellite will not be performed.
3. Capsule Upgrade can now be upgraded later(change in tasks.py) after Satellite upgrade.

### CODE REVIEW

**Comment by jyejare** [comment]
closes #467

**Comment by jyejare** [comment]
@elyezer , Actually this is just a copy of a code from https://github.com/SatelliteQE/robottelo/blob/master/robottelo/host_info.py#L40, which I trusted on. And modified with my way.
Also I tested this code on Sat 6.2 and I didnt realize that this will fail on Sat 6.0

I am looking into that. Thanks for pointing e to this.

**Comment by jyejare** [comment]
@elyezer , The issue should now be resolved, I tested it on my system. Please revisit to this PR.

**Comment by jyejare** [comment]
@SatelliteQE/quality-engineers , Please review this PR, pending from long time !!

File: automation_tools/satellite6/upgrade/__init__.py [hunk 1]
```diff
@@ -21,6 +21,9 @@
     satellite6_upgrade,
     satellite6_zstream_upgrade
 )
+from automation_tools.satellite6.upgrade.tasks import (
+    determine_satellite_version
```
Comment (sghai) [inline automation_tools/satellite6/upgrade/__init__.py]: nitpick: would be good to rename `determine_satellite_version` to  `get_sat_version`

File: automation_tools/satellite6/upgrade/tasks.py [hunk 2]
```diff
@@ -629,3 +641,42 @@ def docker_execute_command(container_id, command, quiet=False):
...
+    else:
+        return 'Unavailable', cmd_result
+
+
+def determine_satellite_version():
+    """Determines and returns the installed Satellite version on system
+
+    :return string: Satellite version
+    """
+    _SAT_6_2_VERSION_COMMAND = u'porpm -q satellite'
```
Comment (sghai) [inline automation_tools/satellite6/upgrade/tasks.py]: `rpm -q satellite` ?

**Comment by sghai** [comment]
ACK.. pending a minor change.

**Comment by jyejare** [comment]
@sghai , I worked on your comment, please have a look again.

**Comment by sghai** [comment]
ACK.. looks good.

@elyezer : we need the changes to test out recently built upgrade images so merging it now.

**Pull request merged**